src/components/DestinationHeroSection.vue

```diff
@@ -0,0 +1,326 @@
+<template>
+  <div class="hero">
+    <h1><span>01</span>Pick your destination</h1>
+    <div class="hero-section">
+      <div>
+        <img :src="defaultImage" alt="" />
+      </div>
+      <div class="options">
+        <ul>
+          <li
+            v-for="planet in planets"
+            :key="planet.name"
+            @click="selectPlanet(planet)"
+            :class="{ selected: planet.name === selectedOption }"
+          >
+            {{ planet.name }}
+          </li>
+        </ul>
+        <div v-for="planet in planets" :key="planet.name">
```

Comment:
There shouldn't be a for loop here. The loop should only be on the li tags.
You also don't need the `v-if` below, but I understand what you're trying to do

Everything below should be `selectedPlanet.name`, `selectedPlanet.description` etc